tutorials/asr/ASR_Context_Biasing.ipynb

```diff
@@ -898,7 +898,7 @@
     "        print(f\"[ref text]: {target_transcripts[idx]}\")\n",
     "    else:\n",
     "        # if no spotted words, use standard greedy predictions\n",
-    "        pred_text = ctc_model.wer.decoding.ctc_decoder_predictions_tensor(greedy_predicts)[0][0]"
+    "        pred_text = ctc_model.wer.decoding.ctc_decoder_predictions_tensor(greedy_predicts)[0]"
```

Comment:
isn't this supposed to be `ctc_model.wer.decoding.ctc_decoder_predictions_tensor(greedy_predicts)[0].text`?